### PR TITLE
changes to the navbar

### PR DESCRIPTION
**Pull request by turbomartine**

### CODE REVIEW

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
-  <nav>
+  <nav class="ma-1 mb-5">
     <!-- logo -->
     <div class="logo__wrapper">
       <LogoKokkos size="lg" class="mr-1"/>
-      <h3>kokkos</h3>
+      <h2>kokkos</h2>
```
Comment (TurboTobias) [inline src/components/Navbar.vue]: Should the size of h3 be adjusted or is this right, to make h3 into h2?

File: src/components/Navbar.vue [hunk 2]
```diff
@@ -35,6 +35,8 @@ nav {
   display: flex;
   align-items: center;
   justify-content: space-between;
+  width: 100%;
+  max-width: 1050px;
```
Comment (TurboTobias) [inline src/components/Navbar.vue]: If this size is used in other places it's smart to store and use it as a variable

File: src/components/Navbar.vue (hunk 1 above)
Comment (turbomartine) [reply src/components/Navbar.vue]: I have done both :)

File: src/components/Navbar.vue (hunk 2 above)
Comment (turbomartine) [reply src/components/Navbar.vue]: I think the size of the nav-bar will be specific, and not to be used elsewhere. But, as I am not sure, you might be correct. I will make a variable

**Pull request merged**